docs/kcMigration/KeycloakMigration.md

```diff
@@ -0,0 +1,50 @@
+---
+description: KeyCloak (RH-SSO) Migration from Pathfinder to BCGov service
+resourceType: Documentation
+title: BC Government KeyCloak (RH-SSO) Service Migration Plan
+---
+
+## KeyCloak Service Migration
+
+Hi team, we are in the process of transforming KeyCloak from a pathfinder service to an organizational BCGov service. For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).
```

Comment:
1. I'd avoid sending this as a personal letter using words like 'we'. Just tell them how it is. 
Something like 'Keycloak is being converted to an BC Gov organizational service. This means it is being migrated from the `pathfinder` service.

refer to [BC Government SSO Service Definition]..